build.sbt

```diff
@@ -64,8 +71,18 @@ val collections =
       ).toList
     )
 
+val collectionsJVM = collections.jvm
+val collectionsJS = collections.js
+
+val root = project.in(file("."))
```

Comment:
Interesting, we can have two sbt projects in the same file? Well, the `collections` project is not really an sbt project since it’s a cross project, that might be the reason why…

Reply:
Indeed, `collections` is *really not* an sbt project. It's just a `val` like any other.